tests/tools/run-bindgen.py

```diff
@@ -1,174 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Yeah. One thing we can't do with Rust-based testing* is run a single header test. As the suite runs so fast now, I think I'll just remove that section.

\* in the absence of data-driven testing that matches against strings passed by `cargo test`…